test/lib/Elastica/Test/ResponseTest.php

```diff
@@ -201,6 +201,23 @@ public function testIsNotOkBulkItemsWithStatusField()
     }
 
     /**
+     * @group unit
+     */
+    public function testDecodeResponseWithBigIntSetToTrue()
+    {
+        $response = new Response(json_encode(array(
+            'took' => 213,
+            'items' => array(
+                array('index' => array('_index' => 'rohlik', '_type' => 'grocery', '_id' => '707891', '_version' => 4, 'status' => 200)),
+                array('index' => array('_index' => 'rohlik', '_type' => 'grocery', '_id' => '707893', '_version' => 4, 'status' => 200)),
+            ),
+        )));
+        $response->setJsonBigintConversion(true);
+
+        $this->assertTrue(is_array($response->getData()));
```

Comment:
No, it wouldn't since the return value of https://github.com/ruflin/Elastica/pull/1005/files#diff-b364251d02e70310f3c6e443372397d3L208 would be `\stdClass` and not an array and there is not conversion or casting happening till the end of method.